src/main.ts

```diff
@@ -100,13 +103,33 @@ const executeDockerCommand = async (dockerCommand: string, continueOnError: bool
     var execOptions: any = {
         outStream: new NullOutstreamStringWritable({ decodeStrings: false }),
         listeners: {
-            stdout: (data: any) => console.log(data.toString()), //to log the script output while the script is running.
-            errline: (data: string) => {
+            stdout: async (data: any) => {
+                let scannedResult = {result: null};
+                if(!config.credScanEnable) console.log(data.toString());

```

Comment:
You can merge the if condition of L108, 109 and L110 will just have else statement.

Reply:
I did this to give priority to the feature flag. So if this is disabled then the user can't enable it from the workflow variable. Is this unnecessary?